tests/testConvolved.py

```diff
@@ -0,0 +1,192 @@
+                                center, scale.asDegrees(), 0.0, 0.0, scale.asDegrees()))
+    return exp, center
+
+
+class ConvolvedFluxTestCase(lsst.utils.tests.TestCase):
+    """A test case for measuring convolved fluxes"""
+
+    def checkSchema(self, schema, names):
+        """Check that the schema includes flux, fluxSigma and flag elements for each measurement
+
+        Parameters
+        ----------
+        schema : `lsst.afw.table.Schema`
+            Schema to check.
+        names : `list` of `str`
+            List of measurement algorithm names
+        """
+        for name in names:
+            self.assertIn(name + "_flux", schema)
+            self.assertIn(name + "_fluxSigma", schema)
+            self.assertIn(name + "_flag", schema)
+
+    def check(self, psfFwhm=0.5, flux=1000.0):
+        """Check that we can measure convolved fluxes
+    
+        We create an image with a Gaussian PSF and a single point source.
+        Measurements of the point source should match expectations for a
+        Gaussian of the known sigma and known aperture radius.
+
+        Parameters
+        ----------
+        psfFwhm : `float`
+            PSF FWHM (arcsec)
+        flux : `float`
+            Source flux (ADU)
+        """
+        bbox = afwGeom.Box2I(afwGeom.Point2I(12345, 6789), afwGeom.Extent2I(200, 300))
+
+        # We'll only achieve the target accuracy if the pixel scale is rather smaller than Gaussians
+        # involved. Otherwise it's important to consider the convolution with the pixel grid, and we're
+        # not doing that here.
+        scale = 0.1*afwGeom.arcseconds
+
+        exposure, center = makeExposure(bbox, scale, psfFwhm, flux)
+        msConfig = measBase.SingleFrameMeasurementConfig()
+        algName = "ext_convolved_ConvolvedFlux"
+        msConfig.plugins.names.add(algName)
+        msConfig.plugins.names.add("ext_photometryKron_KronFlux")
+        values = [ii/scale.asArcseconds() for ii in (0.6, 0.8, 1.0, 1.2)]
+        algConfig = msConfig.algorithms[algName]
+        algConfig.seeing = values
+        algConfig.aperture.radii = values
+        algConfig.aperture.maxSincRadius = max(values) + 1  # Get as exact as we can
+
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        algMetadata = dafBase.PropertyList()
+
+        task = measBase.SingleFrameMeasurementTask(schema, config=msConfig, algMetadata=algMetadata)
+        measCat = afwTable.SourceCatalog(schema)
+        source = measCat.addNew()
+        source.getTable().setMetadata(algMetadata)
+        ss = afwDetection.FootprintSet(exposure.getMaskedImage(), afwDetection.Threshold(0.1))
+        fp = ss.getFootprints()[0]
+        source.setFootprint(fp)
+        task.run(measCat, exposure)
+
+        if display:
+            ds9.mtv(exposure, frame=frame)
+            ds9.dot("x", center.getX() - exposure.getX0(), center.getY() - exposure.getY0(), frame=frame)
+            import pdb;pdb.set_trace()
```

Comment:
This has the advantage that you can test the display code is not crazy with the dummy device. The more code paths we can check the better. Of course, that means that linking debugging to display is not a good plan.